deps/http_parser/http_parser.gyp

```diff
@@ -79,7 +83,7 @@
         'defines': [ 'HTTP_PARSER_STRICT=1' ],
         'include_dirs': [ '.' ],
       },
-      'defines': [ 'HTTP_MAX_HEADER_SIZE=8192', 'HTTP_PARSER_STRICT=1' ],
+      'defines': [ 'HTTP_MAX_HEADER_SIZE=<(http_max_header_size)', 'HTTP_PARSER_STRICT=1' ],
```

Comment:
You should open an upstream PR for this because these changes are going to be wiped out next time http-parser is upgraded.